src/WebJobs.Script/Binding/Manual/ManualTriggerHostBuilderExtensions.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.Azure.WebJobs.Script.Binding;
+using Microsoft.Extensions.Hosting;
+
+namespace Microsoft.Azure.WebJobs.Script
+{
+    public static class ManualTriggerHostBuilderExtensions
+    {
+        public static IHostBuilder AddManualTrigger(this IHostBuilder builder)
+        {
+            return builder.AddExtension<ManualTriggerConfigProvider>();
```

Comment:
This installs the extension and registers it? I thought the "startup" classes for the extensions as I have in my Extensions PR took care of registering them?

Reply:
Yes, the startup classes take care of registration, but for built in extensions like this (this one is actually defined in Script), we just register them directly.